SIRF_data_preparation/NeuroLF_Hoffman_Dataset/BSREM_NeuroLF_Hoffman.py

```diff
@@ -1,9 +1,16 @@
-from petric import MetricsWithTimeout, get_data
+from petric import SRCDIR, MetricsWithTimeout, get_data
```

Comment:
the `SIRF_data_preparation/plot_BSREM_metrics.py` way (using `OUTDIR` too) seems better

Reply:
agreed. I'm putting some of that in  #88 but hard to go through the whole thing.